README.md

```diff
@@ -73,47 +75,40 @@ macOS tools:
 
 Unix tools:
 
-* [Git] for version control
 * [htop] for viewing process information in the CLI
 * [OpenSSL] for Transport Layer Security (TLS)
 * [The Silver Searcher] for finding things in files
-* [vim] as a handy in-terminal editor for rainy days
+* [tmux] for saving project state and switching between projects
 * [Zsh] as your shell
 * [Oh My Zsh] framework for Zsh
 
-[Git]: https://git-scm.com/
-[htop]: https://hisham.hm/htop/
+[htop]: https://htop.dev/
 [OpenSSL]: https://www.openssl.org/
 [The Silver Searcher]: https://github.com/ggreer/the_silver_searcher
-[vim]: https://www.vim.org/
+[tmux]: http://tmux.github.io/
 [Zsh]: http://www.zsh.org/
+[Oh My Zsh]: https://ohmyz.sh/
 
 Development tools:
 
 * [AWS CLI] for interacting with the Amazon API
-* [Heroku CLI] for interacting with the Heroku API
 * [Sentry CLI] for interacting with the Sentry API
 
 [AWS CLI]: https://aws.amazon.com/cli/
-[Heroku CLI]: https://devcenter.heroku.com/articles/heroku-cli
 [Sentry CLI]: https://github.com/getsentry/sentry-cli
-[Oh My Zsh]: https://ohmyz.sh/
 
 Programming languages, package managers, and configuration:
 
-* [Node.js] and [npm], for running apps and installing JavaScript packages. This installer uses `nvm` to manage Node versions.
-* [Ruby] stable for writing general-purpose code. This installer uses `rbenv` to manage Ruby versions.
-* [rbenv] for managing your Ruby environment
-* [Node.js] and [npm], for running apps and installing JavaScript packages
-* [nvm] for managing versions of node.js
-* [Ruby] stable for writing general-purpose code
+* [Node.js] and [npm], for running apps and installing JavaScript packages. This installer uses [nvm] to manage Node versions.
+* [Ruby] stable for writing general-purpose code. This installer uses [rbenv] to manage Ruby versions.
+* [Rosetta 2] for running tools that are not supported in Apple silicon processors
 
 [Node.js]: http://nodejs.org/
 [npm]: https://www.npmjs.org/
 [nvm]: https://github.com/nvm-sh/nvm
 [Ruby]: https://www.ruby-lang.org/en/
 [rbenv]: https://github.com/rbenv/rbenv
-
+[Rosetta 2]: https://developer.apple.com/documentation/apple-silicon/about-the-rosetta-translation-environment
```

Comment:
Thoughtbot added this upstream. At some point this got installed on my machine and is running now so I added it here.